infinitemoney.py

```diff
@@ -1,4 +1,76 @@
-# This generates infinite money.
-while True:
-    print("Money")
-    print("Money")
+# This generates infinite money :3
+
+from random import shuffle, randint, choice
+
+class Letter():
+    def __init__(self):
+        return
+
+    def giveValue(self,value):
+        self.value = value
+       
+        self.letterM = 'M'
+        self.lettero = 'o'
+        self.lettern = 'n'
+        self.lettere = 'e'
+        self.lettery = 'y'
+        
+        if (value == self.letterM):
+            self.value = self.letterM
+        elif (value == self.lettero):
+            self.value = self.lettero
+        elif (value == self.lettern):
+            self.value = self.lettern
+        elif (value == self.lettere):
+            self.value = self.lettere
+        elif (value == self.lettery):
+            self.value = self.lettery
+        
+        return self.value
+
+class Generator():

```

Comment:
this class seems a bit bloated maybe we should split it up into other classes/files etc